examples/api/test/painting/star_border/star_border.0_test.dart

```diff
@@ -0,0 +1,134 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/painting/star_border/star_border.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('main ...', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.StarBorderApp());
+
+    expect(find.descendant(
+      of: find.byType(Scaffold),
+      matching: find.widgetWithText(AppBar, 'StarBorder Example'),
+    ), findsOne);
+
+    expect(find.descendant(
+      of: find.byType(Scaffold),
+      matching: find.byType(example.StarBorderExample),
+    ), findsOne);
+
+    expect(find.descendant(
+      of: find.byType(example.StarBorderExample),
+      matching: find.byType(ListView),
+    ), findsOne);
+
+    expect(find.descendant(
+      of: find.byType(ListView),
+      matching: find.byType(example.ExampleBorder),
+    ), findsExactly(2));
+
+    expect(find.descendant(
+      of: find.byType(ListView),
+      matching: find.byType(Slider),
+    ), findsExactly(6));
+
+    expect(find.descendant(
+      of: find.byType(ListView),
+      matching: find.byWidgetPredicate((Widget pre)=>pre is SelectableText && pre.data!.startsWith('Container(')),
+    ), findsExactly(2));
+
+    final Finder starFinder = find.byType(example.ExampleBorder).first;
+    example.ExampleBorder exampleBorder = tester.widget<example.ExampleBorder>(starFinder);
+
+    // Defaults
+    expect(exampleBorder.border.points, equals(5));
+    expect(exampleBorder.border.innerRadiusRatio, equals(0.4));
+    expect(exampleBorder.border.pointRounding, equals(0.0));
+    expect(exampleBorder.border.valleyRounding, equals(0.0));
+    expect(exampleBorder.border.squash, equals(0.0));
+    expect(exampleBorder.border.rotation, equals(0.0));
+
+    const Offset offset = Offset(10, 0);
+    await tester.drag(find.descendant(
+      of: find.byWidgetPredicate((Widget w)=> w is example.ControlSlider && w.label=='Points'),
```

Comment:
small whitespace nit (here and below):

```suggestion
      of: find.byWidgetPredicate((Widget w) => w is example.ControlSlider && w.label == 'Points'),
```